### ISSUE TITLE
Got "genomicRegionStrandSpecific" error in MalariaMine build with intermine-1.6.9

### ISSUE THREAD

**Issue by nauer** [intermine/intermine#1476]
Probably a bug in **intermine-1.6.9**

This error occurs when switching to the **Regions** section of the malariamine. Does not happen in version **1.6.8**.

*/var/log/tomcat8/catalina.out*
~~~
[ERROR] 04:15:20 (InsertTag.java:doEndTag:926) 
ServletException in '/model/genomicRegionSearchOptionsBase.jsp': javax.servlet.jsp.JspException: javax.servlet.jsp.JspException: No getter method for property: "strandSpecific" of bean: "org.intermine.bio.web.struts.GenomicRegionSearchForm"

org.apache.jasper.JasperException: An exception occurred processing JSP page /model/genomicRegionSearchOptionsBase.jsp at line 135

132:                     </li>
133: 
134:                     <li id="genomicRegionStrandSpecific">
135:                         <html:checkbox property="strandSpecific" value="true" />
136:                         <span>Check this box to perform a strand-specific region search (search <b>+</b> strand if region start&lt;end; search <b>&ndash;</b> strand if region end&lt;start)</span>
137:                     </li>
138: 


Stacktrace:
	at org.apache.jasper.servlet.JspServletWrapper.handleJspException(JspServletWrapper.java:568)
	at org.apache.jasper.servlet.JspServletWrapper.service(JspServletWrapper.java:455)
	at org.apache.jasper.servlet.JspServlet.serviceJspFile(JspServlet.java:401)
	at org.apache.jasper.servlet.JspServlet.service(JspServlet.java:345)
	at javax.servlet.http.HttpServlet.service(HttpServlet.java:725)
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:291)
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:206)
	at org.apache.catalina.core.ApplicationDispatcher.invoke(ApplicationDispatcher.java:721)
	at org.apache.catalina.core.ApplicationDispatcher.doInclude(ApplicationDispatcher.java:584)
	at org.apache.catalina.core.ApplicationDispatcher.include(ApplicationDispatcher.java:523)
	at org.apache.jasper.runtime.JspRuntimeLibrary.include(JspRuntimeLibrary.java:877)
	at org.apache.jasper.runtime.PageContextImpl.doInclude(PageContextImpl.java:675)
	at org.apache.jasper.runtime.PageContextImpl.include(PageContextImpl.java:669)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:57)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:606)
	at org.apache.struts.tiles.TilesUtilImpl.doInclude(TilesUtilImpl.java:129)
	at org.apache.struts.tiles.TilesUtil.doInclude(TilesUtil.java:152)
	at org.apache.struts.tiles.taglib.InsertTag.doInclude(InsertTag.java:764)
	at org.apache.struts.tiles.taglib.InsertTag$InsertHandler.doEndTag(InsertTag.java:896)
	at org.apache.struts.tiles.taglib.InsertTag.doEndTag(InsertTag.java:465)
	at org.apache.jsp.layout_jsp._jspx_meth_tiles_005fget_005f1(layout_jsp.java:1071)
	at org.apache.jsp.layout_jsp._jspService(layout_jsp.java:237)
	at org.apache.jasper.runtime.HttpJspBase.service(HttpJspBase.java:70)
	at javax.servlet.http.HttpServlet.service(HttpServlet.java:725)
	at org.apache.jasper.servlet.JspServletWrapper.service(JspServletWrapper.java:432)
	at org.apache.jasper.servlet.JspServlet.serviceJspFile(JspServlet.java:401)
	at org.apache.jasper.servlet.JspServlet.service(JspServlet.java:345)
	at javax.servlet.http.HttpServlet.service(HttpServlet.java:725)
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:291)
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:206)
	at org.apache.tomcat.websocket.server.WsFilter.doFilter(WsFilter.java:52)
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:239)
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:206)
	at org.apache.catalina.core.ApplicationDispatcher.invoke(ApplicationDispatcher.java:721)
	at org.apache.catalina.core.ApplicationDispatcher.processRequest(ApplicationDispatcher.java:466)
	at org.apache.catalina.core.ApplicationDispatcher.doForward(ApplicationDispatcher.java:391)
	at org.apache.catalina.core.ApplicationDispatcher.forward(ApplicationDispatcher.java:318)
	at org.apache.struts.action.RequestProcessor.doForward(RequestProcessor.java:1083)
	at org.apache.struts.tiles.TilesRequestProcessor.doForward(TilesRequestProcessor.java:295)
	at org.apache.struts.tiles.TilesRequestProcessor.processTilesDefinition(TilesRequestProcessor.java:271)
	at org.apache.struts.tiles.TilesRequestProcessor.internalModuleRelativeForward(TilesRequestProcessor.java:370)
	at org.apache.struts.action.RequestProcessor.processForward(RequestProcessor.java:566)
	at org.apache.struts.action.RequestProcessor.process(RequestProcessor.java:212)
	at org.apache.struts.action.ActionServlet.process(ActionServlet.java:1913)
	at org.apache.struts.action.ActionServlet.doGet(ActionServlet.java:449)
	at javax.servlet.http.HttpServlet.service(HttpServlet.java:618)
	at javax.servlet.http.HttpServlet.service(HttpServlet.java:725)
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:291)
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:206)
	at org.apache.tomcat.websocket.server.WsFilter.doFilter(WsFilter.java:52)
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:239)
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:206)
	at org.intermine.web.filters.SetCharacterEncodingFilter.doFilter(SetCharacterEncodingFilter.java:139)
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:239)
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:206)
	at org.apache.catalina.core.StandardWrapperValve.invoke(StandardWrapperValve.java:219)
	at org.apache.catalina.core.StandardContextValve.invoke(StandardContextValve.java:106)
	at org.apache.catalina.core.StandardHostValve.invoke(StandardHostValve.java:142)
	at org.apache.catalina.valves.ErrorReportValve.invoke(ErrorReportValve.java:79)
	at org.apache.catalina.valves.AbstractAccessLogValve.invoke(AbstractAccessLogValve.java:610)
	at org.apache.catalina.core.StandardEngineValve.invoke(StandardEngineValve.java:88)
	at org.apache.catalina.connector.CoyoteAdapter.service(CoyoteAdapter.java:537)
	at org.apache.coyote.http11.AbstractHttp11Processor.process(AbstractHttp11Processor.java:1081)
	at org.apache.coyote.AbstractProtocol$AbstractConnectionHandler.process(AbstractProtocol.java:658)
	at org.apache.coyote.http11.Http11NioProtocol$Http11ConnectionHandler.process(Http11NioProtocol.java:222)
	at org.apache.tomcat.util.net.NioEndpoint$SocketProcessor.doRun(NioEndpoint.java:1566)
	at org.apache.tomcat.util.net.NioEndpoint$SocketProcessor.run(NioEndpoint.java:1523)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1145)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:615)
	at org.apache.tomcat.util.threads.TaskThread$WrappingRunnable.run(TaskThread.java:61)
	at java.lang.Thread.run(Thread.java:745)
Caused by: javax.servlet.ServletException: javax.servlet.jsp.JspException: javax.servlet.jsp.JspException: No getter method for property: "strandSpecific" of bean: "org.intermine.bio.web.struts.GenomicRegionSearchForm"
	at org.apache.jasper.runtime.PageContextImpl.doHandlePageException(PageContextImpl.java:901)
	at org.apache.jasper.runtime.PageContextImpl.handlePageException(PageContextImpl.java:834)
	at org.apache.jsp.model.genomicRegionSearchOptionsBase_jsp._jspService(genomicRegionSearchOptionsBase_jsp.java:141)
	at org.apache.jasper.runtime.HttpJspBase.service(HttpJspBase.java:70)
	at javax.servlet.http.HttpServlet.service(HttpServlet.java:725)
	at org.apache.jasper.servlet.JspServletWrapper.service(JspServletWrapper.java:432)
	... 71 more
Caused by: javax.servlet.jsp.JspException: No getter method for property: "strandSpecific" of bean: "org.intermine.bio.web.struts.GenomicRegionSearchForm"
	at org.apache.struts.taglib.TagUtils.lookup(TagUtils.java:915)
	at org.apache.struts.taglib.html.CheckboxTag.isChecked(CheckboxTag.java:151)
	at org.apache.struts.taglib.html.CheckboxTag.doStartTag(CheckboxTag.java:124)
	at org.apache.jsp.model.genomicRegionSearchOptionsBase_jsp._jspx_meth_html_005fcheckbox_005f0(genomicRegionSearchOptionsBase_jsp.java:757)
	at org.apache.jsp.model.genomicRegionSearchOptionsBase_jsp._jspx_meth_html_005fform_005f0(genomicRegionSearchOptionsBase_jsp.java:420)
	at org.apache.jsp.model.genomicRegionSearchOptionsBase_jsp.access$0(genomicRegionSearchOptionsBase_jsp.java:331)
	at org.apache.jsp.model.genomicRegionSearchOptionsBase_jsp$Helper.invoke0(genomicRegionSearchOptionsBase_jsp.java:782)
	at org.apache.jsp.model.genomicRegionSearchOptionsBase_jsp$Helper.invoke(genomicRegionSearchOptionsBase_jsp.java:802)
	at org.apache.jsp.tag.webboxarea_tag.doTag(webboxarea_tag.java:276)
	at org.apache.jsp.model.genomicRegionSearchOptionsBase_jsp._jspx_meth_im_005fboxarea_005f0(genomicRegionSearchOptionsBase_jsp.java:326)
	at org.apache.jsp.model.genomicRegionSearchOptionsBase_jsp._jspx_meth_c_005fotherwise_005f0(genomicRegionSearchOptionsBase_jsp.java:290)
	at org.apache.jsp.model.genomicRegionSearchOptionsBase_jsp._jspx_meth_c_005fchoose_005f0(genomicRegionSearchOptionsBase_jsp.java:179)
	at org.apache.jsp.model.genomicRegionSearchOptionsBase_jsp._jspService(genomicRegionSearchOptionsBase_jsp.java:127)
	... 74 more
146674 [http-nio-8080-exec-6] ERROR org.apache.struts.tiles.taglib.InsertTag  - ServletException in '/model/genomicRegionSearchOptionsBase.jsp': javax.servlet.jsp.JspException: javax.servlet.jsp.JspException: No getter method for property: "strandSpecific" of bean: "org.intermine.bio.web.struts.GenomicRegionSearchForm"
org.apache.jasper.JasperException: An exception occurred processing JSP page /model/genomicRegionSearchOptionsBase.jsp at line 135

132:                     </li>
133: 
134:                     <li id="genomicRegionStrandSpecific">
135:                         <html:checkbox property="strandSpecific" value="true" />
136:                         <span>Check this box to perform a strand-specific region search (search <b>+</b> strand if region start&lt;end; search <b>&ndash;</b> strand if region end&lt;start)</span>
137:                     </li>
138: 


Stacktrace:
	at org.apache.jasper.servlet.JspServletWrapper.handleJspException(JspServletWrapper.java:568)
	at org.apache.jasper.servlet.JspServletWrapper.service(JspServletWrapper.java:455)
	at org.apache.jasper.servlet.JspServlet.serviceJspFile(JspServlet.java:401)
	at org.apache.jasper.servlet.JspServlet.service(JspServlet.java:345)
	at javax.servlet.http.HttpServlet.service(HttpServlet.java:725)
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:291)
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:206)
	at org.apache.catalina.core.ApplicationDispatcher.invoke(ApplicationDispatcher.java:721)
	at org.apache.catalina.core.ApplicationDispatcher.doInclude(ApplicationDispatcher.java:584)
	at org.apache.catalina.core.ApplicationDispatcher.include(ApplicationDispatcher.java:523)
	at org.apache.jasper.runtime.JspRuntimeLibrary.include(JspRuntimeLibrary.java:877)
	at org.apache.jasper.runtime.PageContextImpl.doInclude(PageContextImpl.java:675)
	at org.apache.jasper.runtime.PageContextImpl.include(PageContextImpl.java:669)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:57)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:606)
	at org.apache.struts.tiles.TilesUtilImpl.doInclude(TilesUtilImpl.java:129)
	at org.apache.struts.tiles.TilesUtil.doInclude(TilesUtil.java:152)
	at org.apache.struts.tiles.taglib.InsertTag.doInclude(InsertTag.java:764)
	at org.apache.struts.tiles.taglib.InsertTag$InsertHandler.doEndTag(InsertTag.java:896)
	at org.apache.struts.tiles.taglib.InsertTag.doEndTag(InsertTag.java:465)
	at org.apache.jsp.layout_jsp._jspx_meth_tiles_005fget_005f1(layout_jsp.java:1071)
	at org.apache.jsp.layout_jsp._jspService(layout_jsp.java:237)
	at org.apache.jasper.runtime.HttpJspBase.service(HttpJspBase.java:70)
	at javax.servlet.http.HttpServlet.service(HttpServlet.java:725)
	at org.apache.jasper.servlet.JspServletWrapper.service(JspServletWrapper.java:432)
	at org.apache.jasper.servlet.JspServlet.serviceJspFile(JspServlet.java:401)
	at org.apache.jasper.servlet.JspServlet.service(JspServlet.java:345)
	at javax.servlet.http.HttpServlet.service(HttpServlet.java:725)
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:291)
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:206)
	at org.apache.tomcat.websocket.server.WsFilter.doFilter(WsFilter.java:52)
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:239)
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:206)
	at org.apache.catalina.core.ApplicationDispatcher.invoke(ApplicationDispatcher.java:721)
	at org.apache.catalina.core.ApplicationDispatcher.processRequest(ApplicationDispatcher.java:466)
	at org.apache.catalina.core.ApplicationDispatcher.doForward(ApplicationDispatcher.java:391)
	at org.apache.catalina.core.ApplicationDispatcher.forward(ApplicationDispatcher.java:318)
	at org.apache.struts.action.RequestProcessor.doForward(RequestProcessor.java:1083)
	at org.apache.struts.tiles.TilesRequestProcessor.doForward(TilesRequestProcessor.java:295)
	at org.apache.struts.tiles.TilesRequestProcessor.processTilesDefinition(TilesRequestProcessor.java:271)
	at org.apache.struts.tiles.TilesRequestProcessor.internalModuleRelativeForward(TilesRequestProcessor.java:370)
	at org.apache.struts.action.RequestProcessor.processForward(RequestProcessor.java:566)
	at org.apache.struts.action.RequestProcessor.process(RequestProcessor.java:212)
	at org.apache.struts.action.ActionServlet.process(ActionServlet.java:1913)
	at org.apache.struts.action.ActionServlet.doGet(ActionServlet.java:449)
	at javax.servlet.http.HttpServlet.service(HttpServlet.java:618)
	at javax.servlet.http.HttpServlet.service(HttpServlet.java:725)
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:291)
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:206)
	at org.apache.tomcat.websocket.server.WsFilter.doFilter(WsFilter.java:52)
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:239)
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:206)
	at org.intermine.web.filters.SetCharacterEncodingFilter.doFilter(SetCharacterEncodingFilter.java:139)
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:239)
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:206)
	at org.apache.catalina.core.StandardWrapperValve.invoke(StandardWrapperValve.java:219)
	at org.apache.catalina.core.StandardContextValve.invoke(StandardContextValve.java:106)
	at org.apache.catalina.core.StandardHostValve.invoke(StandardHostValve.java:142)
	at org.apache.catalina.valves.ErrorReportValve.invoke(ErrorReportValve.java:79)
	at org.apache.catalina.valves.AbstractAccessLogValve.invoke(AbstractAccessLogValve.java:610)
	at org.apache.catalina.core.StandardEngineValve.invoke(StandardEngineValve.java:88)
	at org.apache.catalina.connector.CoyoteAdapter.service(CoyoteAdapter.java:537)
	at org.apache.coyote.http11.AbstractHttp11Processor.process(AbstractHttp11Processor.java:1081)
	at org.apache.coyote.AbstractProtocol$AbstractConnectionHandler.process(AbstractProtocol.java:658)
	at org.apache.coyote.http11.Http11NioProtocol$Http11ConnectionHandler.process(Http11NioProtocol.java:222)
	at org.apache.tomcat.util.net.NioEndpoint$SocketProcessor.doRun(NioEndpoint.java:1566)
	at org.apache.tomcat.util.net.NioEndpoint$SocketProcessor.run(NioEndpoint.java:1523)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1145)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:615)
	at org.apache.tomcat.util.threads.TaskThread$WrappingRunnable.run(TaskThread.java:61)
	at java.lang.Thread.run(Thread.java:745)
Caused by: javax.servlet.ServletException: javax.servlet.jsp.JspException: javax.servlet.jsp.JspException: No getter method for property: "strandSpecific" of bean: "org.intermine.bio.web.struts.GenomicRegionSearchForm"
	at org.apache.jasper.runtime.PageContextImpl.doHandlePageException(PageContextImpl.java:901)
	at org.apache.jasper.runtime.PageContextImpl.handlePageException(PageContextImpl.java:834)
	at org.apache.jsp.model.genomicRegionSearchOptionsBase_jsp._jspService(genomicRegionSearchOptionsBase_jsp.java:141)
	at org.apache.jasper.runtime.HttpJspBase.service(HttpJspBase.java:70)
	at javax.servlet.http.HttpServlet.service(HttpServlet.java:725)
	at org.apache.jasper.servlet.JspServletWrapper.service(JspServletWrapper.java:432)
	... 71 more
Caused by: javax.servlet.jsp.JspException: No getter method for property: "strandSpecific" of bean: "org.intermine.bio.web.struts.GenomicRegionSearchForm"
	at org.apache.struts.taglib.TagUtils.lookup(TagUtils.java:915)
	at org.apache.struts.taglib.html.CheckboxTag.isChecked(CheckboxTag.java:151)
	at org.apache.struts.taglib.html.CheckboxTag.doStartTag(CheckboxTag.java:124)
	at org.apache.jsp.model.genomicRegionSearchOptionsBase_jsp._jspx_meth_html_005fcheckbox_005f0(genomicRegionSearchOptionsBase_jsp.java:757)
	at org.apache.jsp.model.genomicRegionSearchOptionsBase_jsp._jspx_meth_html_005fform_005f0(genomicRegionSearchOptionsBase_jsp.java:420)
	at org.apache.jsp.model.genomicRegionSearchOptionsBase_jsp.access$0(genomicRegionSearchOptionsBase_jsp.java:331)
	at org.apache.jsp.model.genomicRegionSearchOptionsBase_jsp$Helper.invoke0(genomicRegionSearchOptionsBase_jsp.java:782)
	at org.apache.jsp.model.genomicRegionSearchOptionsBase_jsp$Helper.invoke(genomicRegionSearchOptionsBase_jsp.java:802)
	at org.apache.jsp.tag.webboxarea_tag.doTag(webboxarea_tag.java:276)
	at org.apache.jsp.model.genomicRegionSearchOptionsBase_jsp._jspx_meth_im_005fboxarea_005f0(genomicRegionSearchOptionsBase_jsp.java:326)
	at org.apache.jsp.model.genomicRegionSearchOptionsBase_jsp._jspx_meth_c_005fotherwise_005f0(genomicRegionSearchOptionsBase_jsp.java:290)
	at org.apache.jsp.model.genomicRegionSearchOptionsBase_jsp._jspx_meth_c_005fchoose_005f0(genomicRegionSearchOptionsBase_jsp.java:179)
	at org.apache.jsp.model.genomicRegionSearchOptionsBase_jsp._jspService(genomicRegionSearchOptionsBase_jsp.java:127)
	... 74 more
~~~

**Comment by justinccdev** [comment]
Yes sorry, this is fixed in #1473.  The example Malariamine does not have any region information anyway - if this bug wasn't there you would just see a message to that effect.

**Comment by julie-sullivan** [comment]
This is now fixed in the master branch.
